client/extensions/wp-super-cache/index.js

```diff
@@ -12,6 +12,6 @@ import { settings } from './controller';
 import { Tabs } from './constants';
 
 export default function() {
-	page( '/extensions/wp-super-cache', siteSelection, sites, navigation, settings );
+	page( '/extensions/wp-super-cache', sites, navigation, settings );
```

Comment:
Yup, WFM. And it also seems to resolve an issue I was seeing but hadn't had a chance to look into yet. 👍